stages/10-openebs-upgrade-check/volume-upgrade-jiva-check

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -ex
+
+######################
+##   Prerequisites  ##
+######################
+
+path=$(pwd)
+# Copy kubeconfig
+mkdir ~/.kube
+cp  $path/.gcp/config ~/.kube/config
+
+kubectl get po -n openebs
+
+# Cloning oep repository which contains all the test scripts
+git clone https://$username:$password@github.com/mayadata-io/oep.git
+cd oep
+
+####################################
+##  Sequencing and Running test   ##
+####################################
+
+kubectl create -f litmus/director/openebs-upgrade-jiva/run_litmus_test.yml
+kubectl get pods -n litmus
+
+test_name=volume-upgrade-jiva-check
```

Comment:
Check the test name(`volume-upgrade-jiva-check`) it should be the same as what you have mentioned in the test and not this.

Reply:
We should change the file name also in correspondence with the actual test name.